testing/kuttl/e2e/cluster-start/01-psqlconnect.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: /home/vagrant/postgres-operator/testing/kuttl/script/portforward.sh
+  background: true
+# this sleep is needed when running in cloud environments, may not be needed
+# when running locally
```

Comment:
Can you update the comment to say why the sleep is needed? aiui, in this case the `connectpsql.sh` will fail if the port-forward hasn't started properly and it tries to connect.

Can you add a `TODO` saying to find a better way to make sure that the port-forward has started :pick: ?

Reply:
will do